src/Clap-Core.package/ClapCommandMatch.class/README.md

```diff
@@ -0,0 +1 @@
+I am a class which represents a successful match with a command.This class contains an array of ClapMatch subclass for each successful match of an element of the command.
```

Comment:
1. correct typography: space after period
2. not an array of `ClapMatch` subclasses! an array of matches (i.e. instances of `ClapMatch` and subclasses)